cogs/greetings.py

```diff
@@ -3,22 +3,30 @@
 import os
 from discord.ext import commands
 
+# Class dedicated to greeting new members when joining the server
 class Greetings(commands.Cog):
+
+# Constructor that assigns the welcome message to the specific new member with their username
     def __init__(self, client):
         self.client = client
         self.welcome_message_id = self.load_welcome_message_id()
 
+# Helper function responsible for loading and returning the welcome message
     def load_welcome_message_id(self):
         try:
             with open("txtfiles/welcome_message_id.txt", "r") as file:
                 return int(file.read())
         except FileNotFoundError:
             return None
 
+# Helper function responsible for retaining the welcome message id specific to the user.
+# As well as storing the id to be loaded later if the bot goes offline in the future.
     def save_welcome_message_id(self, message_id):
         with open("txtfiles/welcome_message_id.txt", "w") as file:
             file.write(str(message_id))
 
+# Command function responsible for sending the welcome message on user join where they must react to a
+# terms and conditions message in order to join the server. As well as a guide to RasenBot.
```

Comment:
Nit: Let's keep it in full sentences so the second sentence could be something like...
`... order to join the server. The message also includes a guide to Rasenbot.`